### DISCUSSION TITLE
Container GUI Variations

### DISCUSSION THREAD

**Discussion by waffleeiron** [BedrockTweaks/Files#336] [Ideas]
Add variations for container GUIs to make it look like the respective container when opening it, also blocks like crafting tables, cartograohy tables etc. that can't actually store items. I know it's been done by others before but for compatibility with brewing stand guide and darker UI

**Comment by PAPISOP** [comment]
I didn't understand what you mean by "variated container gui"? like how the variation should be looking for them @waffleeiron

**Reply by IdotIcom** [reply]
Maybe like this

https://mcpedl.com/colourful-containers-be-2/
